ex5.yml

```diff
@@ -20,10 +20,16 @@ pdi:
       - file: ex5-meta${pcoord[0]}x${pcoord[1]}.h5
         write: [ dsize, psize ]
       - file: ex5-data${pcoord[0]}x${pcoord[1]}.h5
-        on_event: ""
-        when: '$ii>0 & ' # `&' is a local `and' operation
+        #*** enable the output upon the event
+        #...
+        #*** enable the output when ii>0 and ii<3 (`&` is a local `and` operation )
```

Comment:
I don't remember why we put "local" here. Now it seems confusing...

Reply:
The solution of ex5.yml instead mentions a "logical" operation, it must be a typographical error.